src/libs/actions/Task.js

```diff
@@ -59,9 +59,10 @@ function clearOutTaskInfo() {
  * @param {String} assigneeEmail
  * @param {Number} assigneeAccountID
  * @param {Object} assigneeChatReport - The chat report between you and the assignee
+ * @param {Number} policyID - Policy ID of the share destination report
  */
-function createTaskAndNavigate(parentReportID, title, description, assigneeEmail, assigneeAccountID = 0, assigneeChatReport = null) {
-    const optimisticTaskReport = ReportUtils.buildOptimisticTaskReport(currentUserAccountID, assigneeAccountID, parentReportID, title, description);
+function createTaskAndNavigate(parentReportID, title, description, assigneeEmail, assigneeAccountID = 0, assigneeChatReport = null, policyID = CONST.POLICY.OWNER_ACCOUNT_ID_FAKE) {
```

Comment:
@Ollyws I just noticed the `policyID` is actually being passed a string. So wouldn't `CONST.POLICY.OWNER_EMAIL_FAKE` be a better choice?

Reply:
@Ollyws Here is an example of setting the `policyID` to `CONST.POLICY.OWNER_EMAIL_FAKE` instead of `CONST.POLICY.OWNER_ACCOUNT_ID`
https://github.com/Expensify/App/blob/f39e93f119c7bb76a56fb7df28251d84a37b65bc/src/libs/ReportUtils.js#L2415-L2420